### ISSUE TITLE
[MISP] Python Exception

### ISSUE THREAD

**Issue by xme** [OpenCTI-Platform/connectors#393]
Please replace every line in curly brackets { like this } with an appropriate answer, and remove this line.

## Description

The connector fails at importing a MISP event.

## Environment

Latest MISP connector & OpenCTI

## Actual Output

```
connector-misp    | INFO:root:MISP returned 6 events.
connector-misp    | INFO:root:Processing event 5ca4718b-7f38-4822-83b7-0a1a0a00b412
connector-misp    | Traceback (most recent call last):
connector-misp    |   File "/opt/opencti-connector-misp/misp.py", line 1454, in <module>
connector-misp    |     mispConnector.run()
connector-misp    |   File "/opt/opencti-connector-misp/misp.py", line 240, in run
connector-misp    |     self.process_events(work_id, events)
connector-misp    |   File "/opt/opencti-connector-misp/misp.py", line 537, in process_events
connector-misp    |     src_result = self.find_type_by_uuid(ref_src, bundle_objects)
connector-misp    |   File "/opt/opencti-connector-misp/misp.py", line 1407, in find_type_by_uuid
connector-misp    |     result = list(filter(l_find, bundle_objects))
connector-misp    |   File "/opt/opencti-connector-misp/misp.py", line 1406, in <lambda>
connector-misp    |     l_find = lambda o: o.id.endswith("--" + uuid)
connector-misp    | TypeError: can only concatenate str (not "NoneType") to str
```

**Comment by nor3th** [comment]
Could you please provide an anonymized version of that MISP event? Preferably in a json format.

**Comment by SamuelHassine** [comment]
Hello @xme ,

I've solved this issue. It will be available in the 5.0.0 version.

Kind regards,
Samuel
